main.go

```diff
@@ -4,37 +4,42 @@ import (
 	"context"
 	"os"
 
-	"github.com/asecurityteam/asset-inventory-api/pkg/domain"
-	"github.com/asecurityteam/asset-inventory-api/pkg/handlers/v1"
+	v1 "github.com/asecurityteam/asset-inventory-api/pkg/handlers/v1"
+	"github.com/asecurityteam/asset-inventory-api/pkg/storage"
 	"github.com/asecurityteam/runhttp"
 	serverfull "github.com/asecurityteam/serverfull/pkg"
 	serverfulldomain "github.com/asecurityteam/serverfull/pkg/domain"
 	"github.com/asecurityteam/settings"
 	"github.com/aws/aws-lambda-go/lambda"
 )
 
-type nopStorage struct{}
-
-func (s *nopStorage) StoreCloudAsset(ctx context.Context, _ domain.CloudAssetChanges) error {
-	runhttp.LoggerFromContext(ctx).Info("store cloud asset stub")
-	return nil
-}
-
 func main() {
 	ctx := context.Background()
+
 	insert := &v1.CloudInsertHandler{
 		LogFn:   runhttp.LoggerFromContext,
 		StatFn:  runhttp.StatFromContext,
-		Storage: &nopStorage{},
+		Storage: &db,
```

Comment:
`db` hasn't been defined at this point. Since main.go isn't included in the unit tests, I usually do a `go build` and a linter check before I push.

Reply:
I thought for sure I had done that, but maybe it was just my local integration test.  I _know_ I did the lint, but it didn't catch it.  Anyway... my bad.